catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergColumn.java

```diff
@@ -41,6 +41,7 @@ protected IcebergColumn internalBuild() {
       icebergColumn.comment = comment;
       icebergColumn.dataType = dataType;
       icebergColumn.nullable = nullable;
+      icebergColumn.defaultValue = defaultValue == null ? DEFAULT_VALUE_NOT_SET : defaultValue;
```

Comment:
The following two SQLs:
```
create table t1(id varchar(20) default null);
```

```
create table t1 (id varchar(20));
```

In the two cases above, the `defaultValue` is null, how can we distinguish whether users have set the default value or not?

Reply:
The case one, the user should use `Lirerals.NULL`; in case two, `DEFAULT_VALUE_NOT_SET` will be filled in, and the result of the column default value is determined by the underlying engine.